src/stores/courseData.js

```diff
@@ -302,10 +306,11 @@ function getMatchingAttributes (gir, hass, ci) {
       offered_summer: accumObject.offered_summer || nextClass.offered_summer,
       offered_IAP: accumObject.offered_IAP || nextClass.offered_IAP,
       offered_fall: accumObject.offered_fall || nextClass.offered_fall,
+      offered_terms: offered_terms || nextClass.offered_terms,
```

Comment:
Not sure this is doing what you want - it looks like offered_terms is a string and || will not combine them properly